viper/src/ast_utils.rs

```diff
@@ -35,15 +35,18 @@ impl<'a> AstUtils<'a> {
     pub fn pretty_print(&self, program: Program<'a>) -> String {
         let fast_pretty_printer_wrapper =
             silver::ast::pretty::FastPrettyPrinter_object::with(self.env);
-        self.jni.get_string(
-            self.jni.unwrap_result(
-                fast_pretty_printer_wrapper.call_pretty(
-                    self.jni
-                        .unwrap_result(fast_pretty_printer_wrapper.singleton()),
-                    program.to_jobject(),
-                ),
-            ),
-        )
+        self.jni.get_string(self.jni.unwrap_result(fast_pretty_printer_wrapper.call_pretty(self.jni
+            .unwrap_result(fast_pretty_printer_wrapper.singleton()), program.to_jobject())))
+        
+        // let s = self.jni.get_string(
+        //     self.jni.unwrap_result(
+        //         fast_pretty_printer_wrapper.call_pretty(
+        //             self.jni
+        //                 .unwrap_result(fast_pretty_printer_wrapper.singleton()),
+        //             program.to_jobject(),
+        //         ),
+        //     ),
+        // );
```

Comment:
```suggestion
        let s = self.jni.get_string(
            self.jni.unwrap_result(
                fast_pretty_printer_wrapper.call_pretty(
                    self.jni
                        .unwrap_result(fast_pretty_printer_wrapper.singleton()),
                    program.to_jobject(),
                ),
            ),
        );
```
What are these changes…